pro/src/components/ImageUploader/ButtonImageEdit/ModalImageEdit/ModalImageCrop/ModalImageCrop.tsx

```diff
@@ -151,7 +151,7 @@ const ModalImageCrop = ({
             <TextInput
               countCharacters
               className={style['modal-image-crop-credit']}
-              label="Crédit image"
+              label="Crédit de l'image"
```

Comment:
pour pinailler, l'autre apostrophe
```suggestion
              label="Crédit de l’image"
```

Reply:
Tu parles de l'apostrophe Amine ou du label `Crédit de l'image`

Reply:
L'apostrophe, pardon.

Reply:
Ah yes ben je merge comme ça si ça vous va et je rajoute un point pour le parking de mardi

Reply:
j'avais cru comprendre que celle plus inclinée était l'apostrophe française ou un truc du genre, mais oui on peut en parler

Reply:
cahud pour qu'on utilise le "demi anneau gauche sinon", pour plus de swag

![image](https://github.com/pass-culture/pass-culture-main/assets/15941528/3c33320b-1eea-414a-9d2d-fb4d1217c9f1)

Reply:
Très stylé en effet haha